build.sh

```diff
@@ -181,9 +202,17 @@ download_core() {
 }
 
 COMMAND="$1"
-XCMODE="$2"
-: ${XCMODE:=xcodebuild} # must be one of: xcodebuild (default), xcpretty, xctool
+BUILD_SETTINGS="$2"
 
+# Use Debug config if command ends with -debug, otherwise default to Release
```

Comment:
Is support for this worth keeping? Did you do this so there would be minimal changes required to CI?